### PR TITLE
multiple pages : Add Korean Translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ko/common/ar.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# ar
+
+> 아카이브로부터 생성, 수정, 추출 (.a, .so, .o)
```
Comment (einverne) [inline pages.ko/common/ar.md]:
```suggestion
> 아카이브로부터 생성, 수정, 추출 (.a, .so, .o).
```
Missing a period here.

File: pages.ko/common/arc.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# arc
+
+> Arcanist: A CLI for Phabricator.
+> 더많은 정보<https://secure.phabricator.com/book/phabricator/article/arcanist/>.
```
Comment (einverne) [inline pages.ko/common/arc.md]:
```suggestion
> 더많은 정보: <https://secure.phabricator.com/book/phabricator/article/arcanist/>.
```

File: pages.ko/common/aria2.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# aria2
+
+>  경량 멀티 프로토콜 및 멀티 소스 명령줄 다운로드 유틸리티입니다.
```
Comment (einverne) [inline pages.ko/common/aria2.md]:
```suggestion
> 경량 멀티 프로토콜 및 멀티 소스 명령줄 다운로드 유틸리티입니다. 
```

File: pages.ko/common/aria2.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# aria2
+
+>  경량 멀티 프로토콜 및 멀티 소스 명령줄 다운로드 유틸리티입니다. 
+> HTTP, HTTPS, FTP, SFTP, BitTorrent와 Metalink를 지원합니다
```
Comment (einverne) [inline pages.ko/common/aria2.md]:
```suggestion
> HTTP, HTTPS, FTP, SFTP, BitTorrent와 Metalink를 지원합니다.
```

File: pages.ko/common/arp.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# arp
+
+>  시스템의 ARP 캐시 표시 및 조작
```
Comment (einverne) [inline pages.ko/common/arp.md]:
```suggestion
>  시스템의 ARP 캐시 표시 및 조작.
```

**Comment by Proscream** [comment]
@einverne thank you for comment! I've made all the corrections, and I'll tell the co-worker.

**Pull request merged**